### PR TITLE
[FrontDoor Management Client] Update WAF and NetworkExperiment SDKs

### PR DESCRIPTION
**Pull request by eschwabe**
- Add exclusions to Web Application Firewall managed rule sets, groups, and rules.
  - Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/7221
- Add create, update, and get operations for Network Experiment profiles and experiments.
  - Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/7370

### CODE REVIEW

**Comment by eschwabe** [comment]
FYI. @jessicl-ms, @Diego-Perez-Botero, @digavino

File: sdk/frontdoor/Microsoft.Azure.Management.FrontDoor/tests/ScenarioTests/NetworkExperimentTests.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+using Microsoft.Azure.Management.FrontDoor.Models;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using Xunit;
+
+namespace FrontDoor.Tests.ScenarioTests
+{
+    public class NetworkExperimentTests
+    {
+        [Fact]
+        public void NetworkExperimentCRUDTest()
```
Comment (Diego-Perez-Botero) [inline sdk/frontdoor/Microsoft.Azure.Management.FrontDoor/tests/ScenarioTests/NetworkExperimentTests.cs]: >NetworkExperimentCRUDTest [](start = 20, length = 25)

Thanks for writing this test!! :)

**Review by Diego-Perez-Botero — APPROVED** [review]
:shipit:

**Comment by isra-fel** [comment]
Hi @eschwabe , could you
- Add a link to the swagger spec review PR(s) in the description
- Include the `.txt` file generated when you run `generate.ps1`[(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)
- Use version `2.0.0` since there are breaking changes
- Update `AssemblyInfo.cs` as well

Thanks

**Comment by Diego-Perez-Botero** [comment]
@eschwabe - Swagger spec review PR for NetworkExperiment is: https://github.com/Azure/azure-rest-api-specs/pull/7370

**Comment by eschwabe** [comment]
@isra-fel Addressed your comments. Could you please review the latest iteration? Thanks!

**Comment by isra-fel** [comment]
> @isra-fel Addressed your comments. Could you please review the latest iteration? Thanks!

The metadata txt was not in the right place, should be `eng/mgmt/mgmtmetadata/`.
Please follow [this guide](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools) to update your build tools then regenreate.

Other changes are good to me. Thanks.

**Comment by eschwabe** [comment]
> > @isra-fel Addressed your comments. Could you please review the latest iteration? Thanks!
> 
> The metadata txt was not in the right place, should be `eng/mgmt/mgmtmetadata/`.
> Please follow [this guide](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools) to update your build tools then regenreate.
> 
> Other changes are good to me. Thanks.

@isra-fel I had an old version of the powershell module installed. The generate code should be fixed now. Thanks!

**Review by isra-fel — APPROVED** [review]

**Pull request merged**